final_prep/annotations.rb

```diff
@@ -16,28 +20,38 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  # shows you the details of your built bear at stored in the Hash
   return built_bear
 end
-
+# Creates 2 instances of the build_a_bear function/method.
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
-
+  # creates a function that takes in 3 integers as arguments
 def fizzbuzz(num_1, num_2, range)
+# creates a for-loop telling the function to run each number within a range
   (1..range).each do |i|
+    # when running the for-loop this if-statement is checking to see if the variable within the range
+    # is divisible by the values given in the num_1 and num_2 arguments.
     if i % num_1 === 0 && i % num_2 === 0
       puts 'fizzbuzz'
+    # if the above statement was not true, the loop will pass to the elsif statement and now the variable is checked only against the
+    # number is the num_1 argument.
     elsif i % num_1 === 0
       puts 'fizz'
+    # same as the above, but the loop will return buzz is the variable is divisible by the argument num_2
```

Comment:
Since I brought up `return` earlier - in this case, the program is using `puts` which is "showing the user" the string of fizz/buzz/whatever - so in this case we don't want to use the term `return` since the `return` keyword isn't being used.